client/src/test/scala/dwolla/cloudflare/FakeCloudflareService.scala

```diff
@@ -20,19 +20,35 @@ import org.http4s.server.middleware.VirtualHost.exact
 import org.http4s.util.CaseInsensitiveString
 
 class FakeCloudflareService(authorization: CloudflareAuthorization) extends Http4sDsl[IO] {
+
```

Comment:
Did you intend to add these blank lines?

Reply:
No. Removing those lines now.